x-pack/plugins/uptime_monitoring/public/lib/manage_angular_lifecycle.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { unmountComponentAtNode } from 'react-dom';
+
+// @ts-ignore angular params
+export const manageAngularLifecycle = ($scope, $route, elem) => {
```

Comment:
This is probably defined in other plugins too, right? We should really look to move this to some common area for all plugins to consume. Any thoughts on this @bmcconaghy?

Reply:
Yes, I agree it'd be fabulous to have this sort of boilerplate shared in a common location. This function and its sibling in `Beats Central Management` are based on similar code in `Index Management`, but a cursory glance at the rest of the plugins reveals that this sort of thing is duplicated more times.

I created https://github.com/elastic/kibana/issues/25643 to track it.

Reply:
I would recommend this code exist inside the framework adapter, or if you prefer a utility folder in the public dir. placing it here is in conflict with the design guides you are otherwise following. But that’s not a must change

Reply:
https://github.com/elastic/kibana/pull/25480/commits/c452de9fab80cbc575620d23f64402fa5b684bf4